frontend/projects/valtimo/case/src/lib/case-inspection/tabs/logs-tab.component.scss

```
@@ -0,0 +1,172 @@
+      font-size: 0.75rem;
+      word-break: break-all;
+    }
+  }
+
+  &__detail-stacktrace {
+    margin: 0;
+    padding: 0.5rem;
+    background-color: var(--cds-background, #ffffff);
+    border: 1px solid var(--cds-border-subtle, #e0e0e0);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)